### DISCUSSION TITLE
GPU驱动和cuda版本问题

### DISCUSSION THREAD

**Discussion by tracy100** [linuxdeepin/developer-center#3474] [Q&A | 问答板块]
请问显卡驱动版本和cuda驱动版本最高支持到那个版本？

**Comment by BLumia** [comment]
你也没说是什么显卡啊...

一般的显卡支持话题的话，可以看看[论坛讨论](https://bbs.deepin.org/post/244248)吧。

**Reply by Decodetalkers** [reply]
> 你也没说是什么显卡啊...
> 
> 一般的显卡支持话题的话，可以看看[论坛讨论](https://bbs.deepin.org/post/244248)吧。

我记得我们的nvidia驱动每次打包没有打包对应的cuda来着。。导致cuda完全不可用，他说的是nvidia吧
